src/OpenSage.Game/Logic/Object/ModelConditionState.cs

```diff
@@ -110,7 +110,7 @@ private void ParseAnimation(IniParser parser)
         public List<ObjectConditionAnimation> ConditionAnimations { get; private set; } = new List<ObjectConditionAnimation>();
         public List<AnimationStateAnimation> Animations { get; private set; } = new List<AnimationStateAnimation>();
         public AnimationMode AnimationMode { get; private set; }
-        public FloatRange AnimationSpeedFactorRange { get; private set; }
+        public FloatRange? AnimationSpeedFactorRange { get; private set; }
```

Comment:
In general I prefer to give it a default value rather than make it nullable. For example:

```suggestion
        public FloatRange AnimationSpeedFactorRange { get; private set; } = new FloatRange(1.0f, 1.0f);
```

That way it keeps the default value here where it belongs, rather than at callsites.

Reply:
fixed